apps/protocol-frontend-e2e/src/support/commands.ts

```diff
@@ -1,33 +1,79 @@
-// ***********************************************
-// This example commands.js shows you how to
-// create various custom commands and overwrite
-// existing commands.
-//
-// For more comprehensive examples of custom
-// commands please read more here:
-// https://on.cypress.io/custom-commands
-// ***********************************************
-
-// eslint-disable-next-line @typescript-eslint/no-namespace
-declare namespace Cypress {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  interface Chainable<Subject> {
-    login(email: string, password: string): void;
-  }
-}
-//
-// -- This is a parent command --
-Cypress.Commands.add('login', (email, password) => {
-  console.log('Custom command example: Login', email, password);
+import { MockExtensionProvider } from "./metamaskMockExt";
+
+Cypress.Commands.add('login', (network, address, COOKIE) => {
+  cy.visit("http://localhost:3000/", {
+    onBeforeLoad(win) {
+        win.ethereum = new MockExtensionProvider(network, address);
+    }
+    });
+  //add cookie into '/siwe/active' request
+  cy.intercept('/siwe/active', req => {
+    req.headers['Cookie'] = COOKIE;
+  });
+  //network requests
+  cy.interceptGQL('POST',
```

Comment:
What does this intercept do?

Reply:
Without it interceptGQL, we will get the errors below. We are able to add cookies and gql endpoints in the intercept as well.

<img width="654" alt="Screen Shot 2022-09-13 at 11 45 03 AM" src="https://user-images.githubusercontent.com/84946242/189985676-bfd66970-cf25-44a8-8359-0bd43f34e982.png">